image_topic: "/camera/rgb/image_raw"
height: 100
width: 100
FPS:
  detection: 11
  classification: 7